yt_dlp/cookies.py

```diff
@@ -122,13 +123,14 @@ def _extract_firefox_cookies(profile, container, logger):
         return YoutubeDLCookieJar()
 
     if profile is None:
-        search_root = _firefox_browser_dir()
+        search_roots = list(_firefox_browser_dirs())
     elif _is_path(profile):
-        search_root = profile
+        search_roots = [profile]
     else:
-        search_root = os.path.join(_firefox_browser_dir(), profile)
+        search_roots = [os.path.join(path, profile) for path in _firefox_browser_dirs()]
+    search_root = ', '.join(map(repr, search_roots))
```

Comment:
Should we raise here `if not search_roots`? the following `FileNotFoundError` won't be very helpful if `search_root` is an empty string

Reply:
`search_roots` will always contain at least one entry (`_firefox_browser_dirs` yields at least one item and `[profile]`)